css/style.css

```diff
@@ -0,0 +1,15 @@
+h2 {
```

Comment:
Can you setup this style via a class selector?
Something like .page_title ?

Reply:
Added to h2 (html doc) class "page_title"

I concluded to this remark that this is a grammatical correction.